lib_eio_linux/eio_linux.ml

```diff
@@ -753,9 +753,11 @@ module Objects = struct
              (should be fixed in Linux 5.14) *)
           await_readable fd
         );
-        let got = read_upto fd chunk max_len in
-        Cstruct.blit chunk_cs 0 buf 0 got;
-        got
+        match read_upto fd chunk max_len with
```

Comment:
The docs for `read_upto` claim that it raises `End_of_file`. Why isn't that happening?

https://github.com/ocaml-multicore/eio/blob/f22bbafeb22924d5ea5efa96bc2ae734342a4049/lib_eio_linux/eio_linux.ml#L468

Reply:
Ah, I think I missed that while reading the code. Eio_luv is implemented a bit differently, it seems. I will close this PR since it is quite redundant then.